### PR TITLE
unit test verification now reports which columns are NULL

### LINKED ISSUE #111: resource.go:157: row at table has an empty column
Currently when testing you can get the following error:

```
    resource.go:157: row at table aws_ec2_ebs_snapshots has an empty column
```

It's very hard to debug this without either knowing which column is empty or at least printing the violating row to debug this manually.

### PR DESCRIPTION
**Pull request by amanenk**
fixes #111

### CODE REVIEW

File: provider/testing/resource.go [hunk 1]
```diff
@@ -126,44 +128,71 @@ func getEnv(key, fallback string) string {
...
 	return res
 }
+func verifyColumnsBySequence(t *testing.T, conn pgxscan.Querier, sequence VerificationSequence, parentID *string) {
+	sq := squirrel.StatementBuilder.
+		PlaceholderFormat(squirrel.Dollar).
+		Select(fmt.Sprintf("json_agg(%s)", sequence.TableName)).
+		From(sequence.TableName)
+	if parentID == nil {
+		// it is a root object
+		sq = sq.Where(squirrel.Eq{"account_id": "testAccount"})
```
Comment (roneli) [inline provider/testing/resource.go:158]: this seems AWS specific, how would this work on other providers?

**Review by roneli — CHANGES_REQUESTED** [review]
I think the new code is not generic enough, needs some rethinking, imo the SQL query should be more verbose in it's return value.

File: provider/testing/resource.go (hunk 1 above)
Comment (amanenk) [reply provider/testing/resource.go]: Yeah. That is a great suggestion.

**Comment by roneli** [comment]
Fixed in #137

**Pull request closed without merging**